### ISSUE TITLE
Enlarged median nerve

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11665]
@evanc1648
Adding new term: 
- ID: HP:6001498
- label: Enlarged median nerve
- parents: [Enlarged peripheral nerve (HP:0012645)](https://hpo.jax.org/app/browse/term/HP:0012645)
- pmid: PMID:40863993
- def: The size (cross-sectional area) of the median nerve is above the upper limit of normal.. This feature may be appreciated by palpation. Ultrasound examination can be used to quantify the cross-sectional area of the median nerve.
- comment: See Figure 3 of PMID:40863993. An enlarged median nerve may be observed with carpal tunnel syndrome and other conditions.
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Thickened median nerve(NONE)
